modules/EvseManager/EvseManager.cpp

```diff
@@ -893,6 +899,11 @@ void EvseManager::ready() {
         }
     });
 
+    // wait for first powermeter value
+    std::unique_lock<std::mutex> lk(powermeter_mutex);
```

Comment:
Doesn't that deadlock if the first power meter value arrives after that lock? Maybe we should make initial_powermeter_value_received an std::atomic_bool instead and remove the locking in the power meter handler?

Reply:
The `this->powermeter_cv.wait_for` releases the mutex so there should not be a deadlock

Reply:
but you don't need the lock after the wait_for returns. Maybe its better to place both in an own scope